stellargraph/core/graph.py

```diff
@@ -126,6 +132,32 @@ class StellarGraph:
             required by some graph models. These are expected to be
             a numeric feature vector for each node in the graph.
 
+        nodes: DataFrame or dict of hashable to DataFrame
+            Features for every node in the graph. Any columns in the dataframe are taken as numeric
+            edge features of type ``dtype``. If there is only one type of node, a DataFrame can be
+            passed directly, and the type defaults to the ``node_type_default`` parameter. Nodes
+            have an ID taken from the index of the dataframe, and they have to be unique across all
+            types.  For nodes with no features, an appropriate DataFrame can be created with
+            ``pandas.DataFrame([], index=node_ids)``, where ``node_ids`` is a list of the node
+            IDs. This must be used with the ``edges`` argument. **Warning**: this is experimental
+            (see warning above for more details).
+
+        edges: DataFrame or dict of hashable to DataFrame
+            An edge list for each type of edges as a Pandas DataFrame containing a source, target
+            and (optionally) weight column (the names of each are taken from the ``source_column``,
+            ``target_column`` and ``edge_weight_label`` parameters). If there is only one type of
+            edges, a DataFrame can be passed directly, and the type defaults to the
+            ``edge_type_default`` parameter. Edges have an ID taken from the index of the dataframe,
+            and they have to be unique across all types. This must be used with the ``nodes``
+            argument. **Warning**: this is experimental (see warning above for more details).
+
+        source_column: str, optional
```

Comment:
It would be helpful to have the default values for `source_column` and `target_column` mentioned. Also documentation is missing for the `edge_weight_label` parameter.

Reply:
The default value is shown in the class header:

![image](https://user-images.githubusercontent.com/1203825/73315299-d3d01580-4283-11ea-8a97-018898eb99ac.png)

I'm mildly against listing information redundantly, especially in string form, because it makes it very easy to become out of date.

> Also documentation is missing for the edge_weight_label parameter.

Ah, good catch; that parameter existed for the NetworkX type so I didn't think to double check it was documented.

Reply:
fair point! the class header is usually the first thing a user looks at anyway